packages/preferences/src/browser/ui/settings-ui-plus-widget.tsx

```diff
@@ -0,0 +1,46 @@
+/********************************************************************************
+ * Copyright (C) 2020 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { postConstruct, injectable, inject } from 'inversify';
+import { WidgetManager, BoxPanel } from '@theia/core/lib/browser';
+import { SettingsUIPlusEditor } from './settings-ui-plus-editor';
+
+@injectable()
+export class SettingsUIPlusWidget extends BoxPanel {
```

Comment:
Could we get rid of `UIPlus` from naming and call `SettingsWidget`, `SettingsContribution` and so on. What does it mean? file names should be aligned

Reply:
This was just the name we developed it under. I'd be happy to change it.